package.json

```diff
@@ -143,6 +144,7 @@
     "@honeybadger-io/webpack": "^1.2.0",
     "@rails/webpacker": "5.2.1",
     "ahoy.js": "^0.3.7",
+    "autoprefixer": "^9.8.6",
```

Comment:
autoprefixer and postcss dependencies are not dev dependencies as they are required for the build in which Heroku only installs dependencies, not devDependencies.